src/types/Config.tsx

```diff
@@ -402,10 +402,15 @@ export const ALL_PAGES: IPage[] = [
   { name: PAGES.COOKIE_POLICY, element: <CookiePolicy /> },
   { name: PAGES.ABOUTPAGE, element: <AboutPage /> },
   {
-    name: PAGES.TECHNICAL_SETUP,
+    name: PAGES.CONNECTOR_MANAGEMENT,
     role: ROLES.TECHNICAL_SETUP_VIEW,
     element: <EdcConnector />,
   },
+  {
+    name: PAGES.TECHNICAL_SETUP,
+    role: ROLES.TECHNICAL_SETUP_VIEW,
+    element: <Redirect path={PAGES.CONNECTOR_MANAGEMENT} />,
+  },
```

Comment:
Minor comment - nothing which blocks the PR:
I suggest an inline comment to be added. This should get removed again with 24.12. Reason, we expect that after that all users which are using bookmarks have switched to the new page. @oyo what do you think?

Reply:
Added

Reply:
okay for me